tsk/fs/btrfs_csum.c

```diff
@@ -0,0 +1,43 @@
+/*
+** The Sleuth Kit
+**
+** Brian Carrier [carrier <at> sleuthkit [dot] org]
+** Copyright (c) 2003-2011 Brian Carrier.  All rights reserved
+**
+** TASK
+** Copyright (c) 2015 Stefan Pöschel.  All rights reserved
+**
+** This software is distributed under the Common Public License 1.0
+*/
+
+/*
+ * Contains the checksum part for Btrfs file system support.
+ *
+ * This is outsourced to plain C due to the CRC API defining its own bool type
+ *  - which collides with the bool of C++
```

Comment:
This is outdated as of PR #3071, which removes that `bool` typedef.

Reply:
Okay. So what should I do?

Reply:
Delete the comment, I think.